yaml2sbml/yaml2sbml.py

```diff
@@ -110,43 +144,55 @@ def _create_compartment(model: sbml.Model):
 
 def _load_yaml_file(yaml_file: str) -> dict:
     """
-    Loads yaml file and returns the resulting dictionary.
+    Load YAML file from a dictionary.
 
     Arguments:
-        yaml_file: SBML model
+        yaml_file: directory YAML model
```

Comment:
Also here, I guess these names could be `yaml_path` (and description changed to e.g. `path to YAML model file`? But would be some effort, fine as-is

https://unix.stackexchange.com/a/131585